src/routes/PetitionRoutes.js

```diff
@@ -1,10 +1,10 @@
 import { Router } from 'express';
-import PetitionController, { deletePetition } from '../controllers/PetitionController';
+import PetitionController from '../controllers/PetitionController';
```

Comment:
Awesome, we can now distract the importation. I had not done it initially but I look forward to do it in the next PR